ovos_utils/skills/templates/active_skill.py

```diff
@@ -0,0 +1,24 @@
+from ovos_utils.waiting_for_mycroft.base_skill import MycroftSkill
+
+
+class ActiveSkill(MycroftSkill):
+    def bind(self, bus):
+        super(ActiveSkill, self).bind(bus)
+        if bus:
+            """ insert skill in active skill list on load """
+            self.make_active()
+
+    def handle_skill_deactivated(self, message=None):
+        """ skill is always in active skill list,
+        ie, converse is always called """
+        self.make_active()
+
+
+class PassiveSkill(ActiveSkill):
```

Comment:
I can see the utility in making this a child of `ActiveSkill`, but should it reside in the same file? `from active_skill import PassiveSkill` seems confusing, tiny though these templates are.

Reply:
do you suggest I rename the top level module or split into 2 files?

Reply:
either would work. whichever feels cleaner.

`from ovos_utils.templates.skill-templates import PassiveSkill` seems sensible, but that's just imo